jsapp/js/envStore.ts

```diff
@@ -122,9 +157,17 @@ class EnvStore {
   constructor() {
     makeAutoObservable(this);
     this.data = new EnvStoreData();
+    this.fetchData();
+  }
 
-    actions.auth.getEnvironment.completed.listen(this.onGetEnvCompleted.bind(this));
-    actions.auth.getEnvironment();
+  async fetchData() {
+    try {
+      const response = await fetchGet<EnvironmentResponse>(ENV_ENDPOINT);
+      this.onGetEnvCompleted(response);
+    } catch (err) {
+      const errorObj = err as FailResponse;
+      handleApiFail(errorObj);
+    }
```

Comment:
This `catch` block is kind of redundant - because `fetchGet()` (without the `notifyAboutError` option set to `false`) already calls `handleApiFail()`. So if I make the `/environment` endpoint error, I see two errors:

![image](https://github.com/kobotoolbox/kpi/assets/7819986/47b979d7-f0bc-4948-bcaf-cb996fb23a59)

Reply:
Right! Pushed a fix :)